src/components/chat-item.tsx

```
@@ -0,0 +1,277 @@
+                </FormItem>
+              )}
+            />
+            <Button size="sm">Save</Button>
+          </fieldset>
+        </form>
+        <span className="text-[10px]">Press ESC to cancel, enter to save</span>
+      </Form>
+    ) : (
+      <div
+        className={cn('text-sm', { 'text-xs opacity-90 italic': deleted })}
+        dangerouslySetInnerHTML={{ __html: content ?? '' }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)